code42_connector.py

```diff
@@ -65,19 +74,45 @@ def _handle_test_connectivity(self, param):
             # Ideally the client instantiation would happen in `initialize()` but that function does not have access
             # to the action, so it cannot effectively report errors to the UI.
             # Fix this with a decorator or something similar.
-            self._client = py42.sdk.from_local_account(self._cloud_instance, self._username, self._password)
-            self._client.users.get_current()
+            self.client.users.get_current()
         except Exception as exception:
             return action_result.set_status(phantom.APP_ERROR, f"Unable to connect to Code42: {str(exception)}")
 
         self.save_progress("Test Connectivity Passed")
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_add_departing_employee(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+        username = param["username"]
+        departure_date = param.get("departure_date")
+        user_id = self._get_user_id(username)
+        self.client.detectionlists.departing_employee.add(user_id, departure_date=departure_date)
+        action_result.update_summary({})
+        status_message = "{} was added to the departing employee list".format(username)
```

Comment:
we can use f-strings here :-)

Reply:
Can we use them everywhere then? I was copying the techniques I had already observed in the app, so I will change them everywhere if that is the case

Reply:
everywhere in this project, yes